.env.test

```diff
@@ -8,4 +8,6 @@ DATABASE_URL=postgresql://postgres:postgres@localhost:5432/test
 VERSION=0.1.0
 METRICS_CRON=*/15 * * * *
 
+GRANULARITY=15T
```

Comment:
I would expect granularity to be given upon model creation. Is this used as the default?

Reply:
I was goin to put this under discussion because different models will mostlikely require different granularities. It's a somehow easy change to add it as a model property, given that the user won't be able to change it after it's assigned.

Reply:
I agree that the granularity has to be set during the creation of the model. My feeling is that - at least for now - the only granularity available should be the daily one but as john put in the effort already, is totally fine.

Reply:
I feel we have to do this on model level.

Reply:
We already had some discussions with John, right now it can work with all the levels (hourly, daily, etc.) so this is not a problem. On the other point, I fully agree that the user has to set it on the model creation session given as an argument.

@momegas the "is_used" flag, for now, seems more appropriate. The "time based" solution was the first thought, but in case the user gives some new data, concerning a past time that has already a created report this would be a problem. 

Maybe we could use the "update_date" in db for each record, but this may seem a bit more complex.